game/player.go

```diff
@@ -310,12 +312,23 @@ func (p *Player) appendActionsForInstant(answer []*Action, card *Card) []*Action
 								for _, index := range c {
 									selected = append(selected, p.game.Lands()[index].Id)
 								}
-								answer = append(answer, &Action{
+								action := &Action{
 									Type:     Play,
 									Card:     card,
 									Selected: selected,
 									Target:   targetCreature.Id,
-								})
+								}
+
+								gJson := p.game.serialized()
+								cloneGame := deserializeGameState(gJson)
+								cloneGame.TakeActionAndResolve(action)
+								gJson = cloneGame.serialized()
+
+								gameStateString := fmt.Sprintf("%s", gJson)
```

Comment:
does this even work? since lands all get different ids, the game state will serialize differently when you bounce different land back to your hand. so this seems like it would do a bunch of work for no impact.